src/Spd.Resource.Repository/BizLicApplication/Mappings.cs

```diff
@@ -24,7 +24,6 @@ public Mappings()
          .ForMember(d => d.spd_businessmanagerphone, opt => opt.MapFrom(s => s.ManagerPhoneNumber))
          .ForMember(d => d.spd_origin, opt => opt.MapFrom(s => (int)ApplicationOriginOptionSet.Portal))
          .ForMember(d => d.spd_payer, opt => opt.MapFrom(s => (int)PayerPreferenceOptionSet.Applicant))
-         .ForMember(d => d.spd_businesstype, opt => opt.MapFrom(s => SharedMappingFuncs.GetBizType(s.BizTypeCode)))
```

Comment:
I think What Carol wants is fe does not need to input bizType. But in repository layer, bizType still needs to be recorded.
So, in manager layer, get info from biz profile, and set those value to Repo layer.

Reply:
What I understood is that this value will be already set when `submit` request is sent, so that's why I ignore it here. Isn't the case?